_posts/0000-01-02-NParsons76.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think I am going to love coding. It's right up my alley

![GitHub Logo](/images/logo.png)
Format: ![Alt Text](url)